decidim-core/app/views/decidim/messaging/conversations/show.html.erb

```diff
@@ -1,15 +1,17 @@
 <%= render layout: "show", locals: { participants: conversation.interlocutors(current_user) } do %>
-  <div id="messages" class="scroll-to-last-message">
+  <div id="messages" class="space-y-6" data-scroll-last-child>
     <% user_grouped_messages.each do |sender, messages| %>
-        <%= render "messages", sender: sender, messages: messages %>
+      <%= render "messages", sender: sender, messages: messages %>
     <% end %>
   </div>
 
-  <% if conversation.with_deleted_users?(current_user) %>
-    <div class="callout warning margin-top-2"><%= t ".deleted_accounts" %></div>
-  <% elsif conversation.accept_user?(current_user) %>
-    <%= render "reply", form: @form, conversation: conversation %>
-  <% else %>
-    <div class="callout warning margin-top-2"><%= t ".not_allowed" %></div>
-  <% end %>
+  <div class="border-t-2 border-background mt-6 py-12">
+    <% if conversation.with_deleted_users?(current_user) %>
+      <%= cell("decidim/announcement", t("decidim.messaging.conversations.show.deleted_accounts"), callout_class: "warning") %>
+    <% elsif conversation.accept_user?(current_user) %>
+      <%= render "reply", form: @form, conversation: conversation %>
```

Comment:
There is this kind of weird jumping effect with the conversation form when I first enter the view:

https://user-images.githubusercontent.com/864340/190984005-ed82f6f4-5af1-40c1-94e6-4feb1c55732b.mp4

What happens here is that:
- The `<textarea>` is first automatically focused
- The textarea is then unfocused using JavaScript (not sure where this happens)
- Then there is a scrolling animation done through JavaScript

Maybe removing the autofocus from the `<textarea>` would help solving this issue if the idea is to scroll the user to the latest message automatically.